plugins/module_utils/mh/base.py

```diff
@@ -31,6 +31,11 @@ def __init__(self, module=None):
     def diff_mode(self):
         return self.module._diff
 
+    def do_raise(self, *args, **kwargs):
+        exc_class = _MHE
+        kwargs = dict((k, v) for k, v in kwargs.items() if k != 'exception_class')
```

Comment:
Do you intentionally want to keep the `exception_class` filter here?

Reply:
No sense at all... I was in a hurry. Further simplified it.